src/Util/StringUtil/StringUtil.js

```diff
@@ -14,6 +14,39 @@ class StringUtil {
 
     return text.replace(urlRegex, `<a href="$1" target="_blank">$1</a>`);
   }
+
+  /**
+   * This coereces the value of a string by casting it to the most plausible
+   * datatype, guessed by the value itself.
+   *
+   * @param {String} string The input string to coerce.
+   * @return {string|boolean|float|object} The coerced value.
```

Comment:
Missing array. It might make sense to simply say `*` (and probably more correct as well)